src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -19,17 +22,32 @@ public void testBaseLine() {
                 .setnInfections(nInfections)
                 .setnHouseholds(3000)
                 .setIters(1)
-                .setnDays(90);
+                .setnDays(90)
+                .setNoOutput();
 
         List<List<DailyStats>> stats = m.run();
 
         int lastTotalInfected = 10;
         for (DailyStats s : stats.get(0)) {
-            Assert.assertEquals(10000, s.getTotalPopulation());
-            Assert.assertTrue(s.getDead() <= s.getRecovered() * 0.1);
-            Assert.assertTrue(s.getDead() + nInfections >= s.getRecovered() * 0.005);
-            Assert.assertTrue(s.getTotalInfected() < lastTotalInfected * 2);
+            assertEquals(10000, s.getTotalPopulation());
+            assertTrue(s.getDead() <= s.getRecovered() * 0.1);
+            assertTrue(s.getDead() + nInfections >= s.getRecovered() * 0.005);
+            assertTrue(s.getTotalInfected() < lastTotalInfected * 2);
             lastTotalInfected = s.getTotalInfected();
         }
+
+        // Check all infections occurred somewhere
+        int totalDailyInfects = nInfections;
+        int totalHouseInfects = 0;
+        int cummulativeI = 0;
+        for (DailyStats s : stats.get(0)) {
+            cummulativeI = s.getTotalInfected() + s.getRecovered() + s.getDead();
+            totalDailyInfects += s.getTotalDailyInfections();
+            totalHouseInfects += s.getHomeInfections();
+            //assertEquals(cummulativeI, totalDailyInfects);
+        }
+        System.out.println(totalHouseInfects);

```

Comment:
Remove system.out statements

Reply:
I think these is in an older version of the code? I don't see it in b47e3a7 (and I remember removing it)